styles/asciidoc.atom-text-editor.less

```diff
@@ -5,55 +5,64 @@
 atom-text-editor, :host {
   .asciidoc {
 
-    .markup.bold {
-      font-weight: bold;
-    }
+    .markup{
+      -webkit-font-smoothing: auto;
```

Comment:
I think we should remove this line. This is something that should be provided by the base editor theme.